reactor-core/src/main/java/reactor/core/publisher/InternalEmptySink.java

```diff
@@ -67,11 +69,9 @@ default void emitError(Throwable error, Sinks.EmitStrategy strategy) {
 					Operators.onErrorDropped(error, currentContext());
 					return;
 				case FAIL_NON_SERIALIZED:
-					throw new IllegalStateException(
-							"Spec. Rule 1.3 - onSubscribe, onNext, onError and onComplete signaled to a Subscriber MUST be signaled serially."
-					);
+					throw new EmissionException(emission);
```

Comment:
actually, the message is not a bad idea either. don't remember if the `EmissionException` has such a constructor but it would make sense to have one